initrd/bin/flash-gui.sh

```diff
@@ -59,7 +80,12 @@ while true; do
             whiptail --title 'ROM Flashed Successfully' \
               --msgbox "${ROM#"/media/"}\n\nhas been flashed successfully.\n\nPress Enter to reboot\n" 0 80
             umount /media
-            /bin/reboot
+            
+            if [ "${CONFIG_BOARD%_*}" = nitropad-nv41 || "${CONFIG_BOARD%_*}" = nitropad-ns51 ]; then
+              /bin/nitropad-shutdown.sh
+            else
+              /bin/reboot
+            fi
```

Comment:
Hmmmm. Why don't you change /bin/reboot instead? I would prefer to look there instead of having ever growing conditional statements in the code calling reboot for different platforms. You might also look into /bin/poweroff and put this condition after tpm shutdown call?

Reply:
clearly a simpler approach, done